coala_quickstart/coala_quickstart.py

```diff
@@ -139,9 +140,8 @@ def main():
     if args.green_mode:
         bear_settings_obj = collect_bear_settings(relevant_bears)
         green_mode(
-            project_dir, ignore_globs, relevant_bears, bear_settings_obj,
-            MAX_ARGS_GREEN_MODE,
-            MAX_VALUES_GREEN_MODE,
+            project_dir, project_files, ignore_globs, relevant_bears,
```

Comment:
add `project_files, ` without modifying other lines.

reordering can be done later.